pkg/config/config_template.yaml

```diff
@@ -63,15 +63,21 @@ api_key:
 # sslkeylogfile: ""
 
 
-## @param min_tls_version - string - optional - default: "1.0" or "1.2"
-## @env DD_MIN_TLS_VERSION - string - optional - default: "1.0" or "1.2"
+## @param min_tls_version - string - optional - default: "tlsv1.0" or "tlsv1.2"
```

Comment:
In view of deprecation, I think it would be fair to say that the default is just `tlsv1.0`. WDYT?
```suggestion
## @param min_tls_version - string - optional - default: "tlsv1.0"
```